db/migrate/20180330002910_create_spina_events_sessions.rb

```diff
@@ -0,0 +1,12 @@
+class CreateSpinaEventsSessions < ActiveRecord::Migration[5.1]
+  def change
+    create_table :spina_events_sessions do |t|
+      t.integer :conference_id
+      t.string :title
+      t.text :description
+      t.datetime :start_at
+      t.datetime :end_at
```

Comment:
I think this should be duration. It’s easier to update an agenda when you only have to worry about the duration and start time of the conference. The UI is built by “position”